Trustev.WebAsync/ApiClient.cs

```diff
@@ -891,6 +904,8 @@ public static async Task<string> GetTokenAsync()
         /// <returns></returns>
         public static async Task SetTokenAsync()
         {
+
```

Comment:
@mnevinTransunion Minor thing, but I'd remove these extra new lines.

Reply:
sorted that now